src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -1597,6 +1598,87 @@ public static IEnumerable<IEdmStructuralProperty> Key(this IEdmEntityType type)
         }
 
         /// <summary>
+        /// Gets the declared alternate keys of the most defined entity with a declared key present.
+        /// </summary>
+        /// <param name="type">Reference to the calling object.</param>
+        /// <param name="model">The model to be used.</param>
+        /// <returns>Key of this type.</returns>

```

Comment:
/// <returns>Key of this type.</returns>  
=>
/// <returns>Alternate Keys of this type.</returns>